srcpkgs/libchamplain/template

```diff
@@ -26,7 +26,7 @@ esac
 
 libchamplain-devel_package() {
 	depends="${sourcepkg}>=${version}_${revision} libglib-devel cairo-devel
-	 clutter-devel sqlite-devel libsoup-devel"
+	 clutter-gtk-devel sqlite-devel libsoup-devel"
```

Comment:
the pkgconfnig file calls only for clutter-1.0

Reply:
https://travis-ci.org/void-linux/void-packages/jobs/501311805

Reply:
From `meson-log.txt`:

```
Dependency champlain-0.12 found: YES 0.12.19
Determining dependency 'champlain-gtk-0.12' with pkg-config executable '/usr/bin/pkg-config'
Called `/usr/bin/pkg-config --modversion champlain-gtk-0.12` -> 0
0.12.19
Called `/usr/bin/pkg-config --cflags champlain-gtk-0.12` -> 1
```

Reply:
Please also add gtk+3-devel

Reply:
```
$ xpcdeps champlain-gtk-0.12
cairo-devel -> cairo
clutter-devel -> clutter-1.0
clutter-gtk-devel -> clutter-gtk-1.0
gtk+3-devel -> gtk+-3.0
libchamplain-devel -> champlain-0.12
libglib-devel -> glib-2.0
libglib-devel -> gobject-2.0
```

Reply:
done